otsdaq-mu2e/FEInterfaces/DTCFrontEndInterface_interface.cc

```diff
@@ -299,7 +333,11 @@ void DTCFrontEndInterface::registerFEMacros(void)
 					std::vector<std::string>{},
 					std::vector<std::string>{						
 						"Upstream Rx Lock Loss Count"},
-					1);  // requiredUserPermissions
+					1, 
+					"*", 
+					"Displays the number of times the Jitter Attenuator PLL lost lock since the last reset. "
```

Comment:
This should be the number of the times the CFO Control Link lost CDR lock I believe.

Reply:
You are right. This has been updated